08.decoding.process.md

```diff
@@ -433,11 +428,11 @@ if ( clip_to_restricted_range ) {
 }
 ScalingShift = grain_scaling_minus_8 + 8
 for ( y = 0; y < ( (h + subY) >> subY) ; y++ ) {
-  for ( x = 0; x < ( (w + subX) >> subX) ; x++ ) {
-    lumaX = x << subX
-    lumaY = y << subY
+  for ( x = 0; x < ( (w + SubX) >> SubX) ; x++ ) {
+    lumaX = x << SubX
+    lumaY = y << SubY
```

Comment:
The capitalization is inconsistent with the one used three lines above. Also, many other variables start with the small letter. IMO, we should correct the capitalization of the variables in the syntax table rather than changing it throughout the rest of the document.